ci/builder/Dockerfile

```diff
@@ -97,6 +97,7 @@ RUN apt-get update --fix-missing && TZ=UTC DEBIAN_FRONTEND=noninteractive apt-ge
     gdb \
     git \
     gnupg2 \
+    graphvix \
```

Comment:
I wasn't clear to me if this should be added to this block of `apt-get install` or the one above on lline 22.